libs/design/scss/daff-theme.scss

```diff
@@ -1,24 +1,18 @@
 // @docs
-//
-// `daff-theme` is a utility library for themeing the daffodil
+
+// `daff-theme` is a utility library for theming the daffodil
 // design library. It contains many useful mixins, functions,
 // and variables.
-//
+
 // @note You can import this as often as you like, as it only contains SCSS
 // variables, mixins, and functions; thus it won't effect your final
 // compiled stylesheet size.
-//
+
 // @usage
 // ```
-// @import "daff-theme";
+// @import '@daffodil/design/scss/daff-theme';
```

Comment:
Should this doc be an `@import`?

Reply:
I shouldn't have changed this doc because it's not part of this PR. Going to revert it.